lib/xero_gateway/invoice.rb

```diff
@@ -212,7 +213,7 @@ def self.from_xml(invoice_element, gateway = nil, options = {})
           when "Contact" then invoice.contact = Contact.from_xml(element)
           when "Date" then invoice.date = parse_date(element.text)
           when "DueDate" then invoice.due_date = parse_date(element.text)
-          when "UpdatedDateUTC" then invoice.updated_date_utc = parse_date(element.text)
+          when "UpdatedDateUTC" then invoice.updated_at = parse_date_time(element.text)
```

Comment:
Changing this probably makes the PR a breaking change unfortunately. Do you think it would be worth maintaining the `updated_date_utc` API (i.e aliasing it?)

Reply:
well the reason for the change was simply that the attr_accessor is for `updated_at` and not `updated_at_utc`.